oauth2client/contrib/gce.py

```diff
@@ -85,32 +138,128 @@ class AppAssertionCredentials(AssertionCredentials):
     information to generate and refresh its own access tokens.
     """
 
-    @util.positional(2)
-    def __init__(self, scope='', **kwargs):
+    def __init__(self,
+                 scope=None,
+                 service_account_email='default',
```

Comment:
Also would need to change `_get_service_account_info` to use `.get('email', 'default')`